extensions/azurehybridtoolkit/notebooks/hybridbook/content/hadr/configure-failover.ipynb

```diff
@@ -37,25 +37,23 @@
                 "- One or more domain joined VMs in Azure running SQL Server 2016 (or later) in the _same_ availability set or _different_ availability zones that have been registered with the SQL IaaS Agent extension\n",
                 "- Domain joined VMs must have either a \"Enterprise\" or \"Developer\" image SKU to create a cluster in this notebook\n",
                 "\n",
-                "Tutorial on how to set up a domain with VMs can be found [here.](https://docs.microsoft.com/en-us/azure/azure-sql/virtual-machines/windows/availability-group-manually-configure-prerequisites-tutorial)\n",
+                "Tutorial on how to set up a domain with VMs can be found [here](https://docs.microsoft.com/en-us/azure/azure-sql/virtual-machines/windows/availability-group-manually-configure-prerequisites-tutorial) (only follow the steps up to setting the SQL Server Service Accounts)\n",
                 "\n",
                 "Here is a listing of variables that will be used to create the failover cluster:\n",
                 "\n",
                 "| Line # | Name | Example | Description |\n",
                 "| --- | --- | --- | --- |\n",
```

Comment:
Add $Subscription variable to specify the subscription. We want notebooks to use explicit subscription variables to avoid making resources on the "default" sub in case the customer has multiple.

Reply:
Refer to other notebooks that check the subscription variable and add this as a block to set the AzContext: 
if (!$Subscription)
{
    $Subscription = Get-AzSubscription | Out-GridView -PassThru
} 
Set-AzContext -SubscriptionName $Subscription
Connect-AzAccount -Subscription $Subscription